dev/fattest.simplicity/src/componenttest/custom/junit/runner/TestNameFilter.java

```diff
@@ -30,9 +34,42 @@
         FAT_TEST_QNAME = System.getProperty("fat.test.qualified.name");
         if (FAT_TEST_QNAME != null) {
             Log.info(TestNameFilter.class, "<clinit>", "Running only test method with qualified name: " + FAT_TEST_QNAME);
-            int indx = FAT_TEST_QNAME.lastIndexOf('.');
-            FAT_TEST_CLASS = FAT_TEST_QNAME.substring(0, indx);
-            FAT_TEST_METHOD = FAT_TEST_QNAME.substring(indx + 1);
+
+            List<String> classes = new ArrayList<String>();
+            List<String> methods = new ArrayList<String>();
+
+            // foo.bar.ClassA.TestA and
+            // bar.foo.ClassB.TestB will become:
+            // FAT_TEST_CLASS=foo.bar.ClassA,bar.foo.ClassB
+            // FAT_TEST_METHOD=TestA,TestB
+            // This means that foo.bar.ClassA.TestB and bar.foo.ClassB.TestA will match
+            // This doesn't seem very likely, but is possible
+            for (String s : FAT_TEST_QNAME.split(",")) {
+                int indx = s.lastIndexOf('.');
+                classes.add(s.substring(0, indx));
+                methods.add(s.substring(indx + 1));
+            }
+
+            String classList = new String();
+            for (String clazz : classes) {
+                if (classList.length() > 0) {
+                    classList += "," + clazz;
+                } else {
+                    classList = clazz;
+                }
+            }
+
+            String methodList = new String();
+            for (String method : methods) {
+                if (methodList.length() > 0) {
+                    methodList += "," + method;
+                } else {
+                    methodList = method;
+                }
+            }
+
+            FAT_TEST_CLASS = classList;
```

Comment:
I don't think it's necessary to check for comma-separated list for `FAT_TEST_QNAME`, since the property seems to imply that you must supply the fully qulaified name of the test class and method, doesn't seem likely that people will want to do multiples here